daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -641,14 +641,28 @@ func TestCollectBatchMultilinePatternMaxEventAge(t *testing.T) {
 	})
 
 	// Fire ticker batchPublishFrequency seconds later
-	ticks <- time.Now().Add(batchPublishFrequency * time.Second)
+	ticks <- time.Now().Add(batchPublishFrequency + time.Second)
```

Comment:
`batchPublishFrequency` is already a `time.Duration` (see line 40 in cloudwatchlogs.go), so the existing code was an error.